velox/docs/functions/spark/json.rst

```diff
@@ -0,0 +1,13 @@
+==============
+Json Functions
+==============
```

Comment:
nit: perhaps follow https://github.com/facebookincubator/velox/blob/main/velox/docs/functions/presto/json.rst?plain=1#L2 to capitalize JSON.

Reply:
Done, thanks